internal/cli/main.go

```diff
@@ -190,8 +191,23 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 			log.fatalErr(migraterErr)
 		}
 
-		limit := -1
-		if flag.Arg(1) != "" {
+		limit := 0
+		if flag.Arg(1) == "" {
+			fmt.Println("Are you sure you want to apply all down migrations? [y/N]")
```

Comment:
Use `log.Println()` since that's what all the commands are using. It does seem odd to use logging for a prompt though...

Reply:
switched